templates/kubernetes/what-is-kubernetes.html

```diff
@@ -22,9 +22,8 @@ <h1>
           We asked Devs, DevOps and businesses to tell us how they are using Kubernetes, and it is a fascinating read.
         </p>
         <p>
-          <a class="p-button--positive p-link--external" href="http://juju.is/cloud-native-kubernetes-usage-report-2021">
-            Read our findings
-          </a>
+          <a class="p-link--external" href="http://juju.is/cloud-native-kubernetes-usage-report-2021">Read our findings&nbsp;&rsaquo;</a>
```

Comment:
As it's an external link we don't use the little chevron
```suggestion
          <a class="p-link--external" href="http://juju.is/cloud-native-kubernetes-usage-report-2021">Read our findings</a>
```

Reply:
Fixed!